README.md

```diff
@@ -44,7 +44,7 @@ Read more at [hasura.io](https://hasura.io) and the [docs](https://hasura.io/doc
 **Table of Contents**
 
 - [Quickstart:](#quickstart)
-    - [One-click deployment on Heroku](#one-click-deployment-on-heroku)
+    - [One-click deployment on Hasura Cloud](#one-click-deployment-on-hasura-cloud)
     - [Other deployment methods](#other-deployment-methods)
```

Comment:
We also need this one no?
https://github.com/hasura/graphql-engine#other-one-click-deployment-options